src/ddccontrol/main.c

```diff
@@ -189,7 +194,7 @@ int main(int argc, char **argv)
 		"This program comes with ABSOLUTELY NO WARRANTY.\n"
 		"You may redistribute copies of this program under the terms of the GNU General Public License.\n\n"), VERSION);
 	
-	while ((i=getopt(argc,argv, "hdr:w:csfvpb:i:")) >= 0)
+	while ((i=getopt(argc,argv, "hdr:w:csfvpbo:i:")) >= 0)
```

Comment:
Breaks option `b`, because it misses `:`. Also, forgot to change to `l` from `o`. It doesn't work at all. Did you test it?

Correct string would be `hdr:w:csfvpb:i:l:`, ie. just add `l:`. It works then. See: https://www.gnu.org/software/libc/manual/html_node/Using-Getopt.html#Using-Getopt